std/digest/crc.d

```diff
@@ -421,7 +421,7 @@ struct CRC(uint N, ulong P) if (N == 32 || N == 64)
                          "1234567890123456789012345678901234567890");
     assert(digest == cast(ubyte[]) hexString!"bd3eb7765d0a22ae");
 
-    enum ubyte[8] input = hexString!"c3fcd3d7efbeadde";
+    enum ubyte[8] input = cast(ubyte[8]) hexString!"c3fcd3d7efbeadde";
```

Comment:
Just wondering if it is mandatory to use `enum ubyte[8] input`? `auto input` would look idiomatic.